### PR TITLE
added soql desc

### PR DESCRIPTION
**Pull request by AlexTeusz**
Hi I added the missing SOQLQuery description.

### CODE REVIEW

File: modules/salesforce/README.md [hunk 1]
```diff
@@ -18,6 +18,33 @@ If the loginUrl is stored in the secret, Salesforce login to this url, e.g. 'htt
 
   
 
+# Node: soqlQuery 
+
+Takes a **soql query string** and searches for the results in **Salesforce**. An example query would be the following: 
+
+```sql
```
Comment (mastasky) [inline modules/salesforce/README.md]: SOQL is not SQL. Is this a valid SOQL query? Please test it.

Please also change comment and variable names.

Comment (AlexTeusz) [reply modules/salesforce/README.md]: SELECT Firstname FROM CONTACT is working. Even if it is no SQL you can use SELECT.

File: modules/salesforce/README.md [hunk 2]
```diff
@@ -18,6 +18,33 @@ If the loginUrl is stored in the secret, Salesforce login to this url, e.g. 'htt
...
+SELECT Firstname FROM CONTACT
+```
+
+The output will be a JSON that includes all firstnames from your Salesforce contacts: 
+
+```json
+"sql": {
```
Comment (mastasky) [inline modules/salesforce/README.md]: Please change this variable, as it is confusing (it is not SQL)

Comment (AlexTeusz) [reply modules/salesforce/README.md]: done

File: modules/salesforce/README.md [hunk 3]
```diff
@@ -18,6 +18,33 @@ If the loginUrl is stored in the secret, Salesforce login to this url, e.g. 'htt
 
   
 
+# Node: soqlQuery 
+
+Takes a **soql query string** and searches for the results in **Salesforce**. An example query would be the following:
```
Comment (mastasky) [inline modules/salesforce/README.md]: soql capitalized

Comment (mastasky) [reply modules/salesforce/README.md]: Like SOQL

Comment (mastasky) [reply modules/salesforce/README.md]: Link to https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql.htm

**Pull request merged**